### DISCUSSION TITLE
Is there any plan to develop whatsapp client using wry like whatsapp-web.js did with puppeteer

### DISCUSSION THREAD

**Discussion by yan-ad** [tauri-apps/wry#948] [Ideas]
I'm just using whatsapp-web.js for a while, but the main cons for that library is using puppeteer which implement chromium engine which consume lot of memory.

**Comment by FabianLars** [comment]
This is nothing the tauri team will do but there is nothing stopping you or the community from doing it. All the necessary functionality should be there already, and if not, feature requests and pull requests are always appreciated :)
